### PR TITLE
Restructure homepage

### PR DESCRIPTION
**Pull request by espimadrazo**
First stage on restructure of the homepage, next is to program the drop-down menu on the header with JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,228 +1,70 @@
...
-        <a href="about/index.html">About</a>
-      </div>
-      <div>  
-        <input type="search"></input>
-        <button>Search</button>
+          alt="Drawing of an owl"
+          >
+        </a>
       </div>
+      <menu class="index"
```
Comment (mauriciopasquier) [inline index.html]: You didn't close the tag here.

File: index.html [hunk 2]
```diff
@@ -1,228 +1,70 @@
...
+        </ul>  
+      </menu>
     </header>
+ 
+    <div class="line"></div>
 
     <main>
       <h1>Paradox Collection</h1>
       <h2>A personal list of curious psychological phenomena, paradoxes and cognitive biases</h2>
     </main>
```
Comment (mauriciopasquier) [inline index.html]: I would say `<main>` should represent the main content of each page and thus wrap everything that's not header and footer (in this implementation, the "line" divs are part of the header and footer so keep them out of `<main>`)

File: about/index.html [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
 
-<head>
-  <meta charset="UTF-8">  
-  <meta name="viewport" content="width=device-width, initial scale=1">
-  <title>Welcome to my Paradox Collection!</title>
-</head>
+  <head>
+    <meta charset="UTF-8">  
+    <meta name="viewport" content="width=device-width, initial scale=1">
+    <link rel="stylesheet" href="style.css">
```
Comment (mauriciopasquier) [inline about/index.html]: Each subpage should reference the same style.css from the homepage, if you need specific styling on a page you can add a new stylesheet overwriting some of that but keep `../style.css` loaded. And probably you can add custom, per subpage styles there too.

**Review by mauriciopasquier — CHANGES_REQUESTED** [review]
The main thing I would change is the duplication of stylesheets when having a single one at the root and called from every page is much more maintainable.

File: about/index.html (hunk 3 above)
Comment (espimadrazo) [reply about/index.html]: It's really annoying that I can't answer your first comment directly with a general response :p  
Anyway, checkout the latest commit, I fixed everything you pointed out. Next I'll try and see if I can solve the index feature without using JS.

Comment (mauriciopasquier) [reply about/index.html]: Something, something, blame it on push force :P

**Comment by mauriciopasquier** [comment]
Actually, about general comments: the form is down here!

**Review by mauriciopasquier — APPROVED** [review]

**Pull request merged**